manifests/Hyperhue-Studios.yaml

```
@@ -0,0 +1,13 @@
+name: HyperHue-Studios
+authors: ChemtrailOperator
+homepage: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/tree/main
```

```suggestion
homepage: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky
```
don't need that suffix, unless you plan to switch you default branch to something else i guess